generators/entity-server/templates/src/test/gatling/user-files/simulations/EntityGatlingTest.scala.ejs

```diff
@@ -119,6 +126,7 @@ class <%= entityClass %>GatlingTest extends Simulation {
         .get("/login")
         .check(status.is(302))
         .check(header("Location").saveAs("loginUrl"))).exitHereIfFailed
+        .pause(2)
```

Comment:
all these sleep are necessary, otherwise, you'll have some timeout the 1st time you launch gatling